paper.md

```diff
@@ -0,0 +1,144 @@
+---
+title: 'CMinx: A CMake Documentation Generator'
+tags:
+  - Python
+  - CMake
+  - Documentation
+authors:
+  - name: Branden Butler
+    equal-contrib: true
+    affiliation: "1, 2"
+  - name: Ryan M. Richard
+    orcid: 0000-0003-4235-5179
+    equal-contrib: false
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: "1, 2"
+affiliations:
+ - name: Ames National Laboratory, Ames, IA, USA
+   index: 1
+ - name: Iowa State University, Ames, IA, USA
+   index: 2
+date: August 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+This manuscript introduces CMinx, a program for generating
+application programming interface (API) documentation for CMake modules
+(code written in the CMake language). Since most of CMinx's intended audience
+is comprised of C/C++ developers, CMinx is designed to operate similar to
+Doxygen [@doxygen] (the *de facto* C/C++ API documentation tool).
+Specifically, developers annotate their CMake source with documentation
+comments (traditional CMake block comments, starting with an extra "`[`"
+character). The documentation comments, written in reST [@rest],
+describe to the reader how the functions, parameters, and variables should be
+used. Running CMinx on the annotated source code generates reST files containing
+the API documentation. The reST files can then be converted into static
+websites with tools such as Sphinx [@sphinx] or
+easily converted to another format via Pandoc [@pandoc].
+
+Unlike other solutions for documenting CMake modules, CMinx knows the CMake
+language's grammar. This enables CMinx to automatically extract function/macro
+signatures (even when functions are not documented). CMinx also integrates
+seamlessly into existing CMake build systems. CMinx's output is highly
+customizable and easily controlled via a YAML [@yaml] configuration file.
+CMinx has already proved to be an invaluable
+productivity tool in the authors' other projects, and, given that other
+scientific software projects also rely heavily on source code written in CMake,
+we anticipate CMinx will prove invaluable to many additional projects as well.
+
+# Statement of need
+
+The exascale-era of high-performance scientific computing arguably started when
+the Frontier supercomputer achieved a performance of more than 1.1 exaFLOPs
+(10$^{18}$ floating-point operations per second) on the High-Performance
+Linpack Benchmark [@frontier]. With the breakdown of Dennard scaling
+(*i.e.*, the power usage of a transistor is proportional to its area),
+achieving exascale performance is presently only possible by relying
+on heterogeneous hardware. Unfortunately, hardware heterogeneity significantly
+complicates the process of developing high-performance scientific software.
+For many high-performance software packages, these complications start in the
+build system.
```

Comment:
I find that this paragraph to be superfluous. The issue of maintaining build system code has little to do with exascale computing per se. The actual connection is done in the following paragraph, so I would suggest to remove this paragraph entirely. I think the paper actually works well without it.


```suggestion
```